xml/bp_troubleshooting_logging.xml

```diff
@@ -153,14 +153,15 @@
       file every 30 minutes.
     </para>
  </sect1>
- <sect1 xml:id="bp-troubleshooting-valgrind">
-  <title>Valgrind</title>
+ <sect1 xml:id="bp-troubleshooting-memory-util">
+  <title>Monitoring Memory Utilization</title>
     <para>
-      Debugging may also require you to track down memory and threading issues.
-      You can run a single daemon, a type of daemon, or the whole cluster
-      with Valgrind. You should only use Valgrind when developing or
-      debugging Ceph. Valgrind is computationally expensive, and will
-      slow down your system otherwise. Valgrind messages are logged to stderr.
+      To monitor the &ceph; processes memory utilization, pay attention to the
```

Comment:
```suggestion
      To monitor the memory utilization of the &ceph; processes, pay attention to the
```

Reply:
Either this or "Ceph processes' memory allocation" -- but given that it's followed by another possessive, I think this way is clearer.